app/src/main/java/com/example/albumsapp/viewmodel/AlbumViewModel.kt

```
@@ -22,6 +24,7 @@ class AlbumViewModel : ViewModel() {
             try {
                 _albums.value = AlbumsApi.retrofitService.getAlbums()
             } catch (e: Exception) {
+
```

please don't ignore exceptions, instead show user the error